subprojects/base-services/src/main/java/org/gradle/api/JavaVersion.java

```diff
@@ -44,6 +44,14 @@
     @Incubating
     VERSION_12,
 
+    /**
+     * Java 13 major version.
+     *
+     * @since 5.7
```

Comment:
will there be an 5.7?

Reply:
no, but this needs to be 5.7 until we update to 6.0